### PR TITLE
Feature/god mode

### PR DESCRIPTION
**Pull request by DJSteins**
Adds a god mode inventory to the flat map for giving players powerups.

### CODE REVIEW

File: Assets/_PartyParkade/Script/Data/GodMode.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/Data/GodMode.cs:1]: missing Header

File: Assets/_PartyParkade/Script/Data/GodMode.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+using _PartyParkade.Script.Data;
+using System;
+
+public class GodMode : SerializedMonoBehaviour
+{
+    /// <summary>
+    /// Initialize an invetory for each player
+    /// </summary>
+    [TableList]
+    public GodInventory[] inventories = new GodInventory[]
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/Data/GodMode.cs:15]: inventories should be capitalized

**Comment by Gursaanj** [comment]
[Debug mode for Flat Scene](https://trello.com/c/mFtedbld/504-debug-mode-for-flat-scene)

**Comment by DJSteins** [comment]
> What gursaanj said. And there are some spelling mistkaes on "invetory"

the irony in this comment is amazing

**Pull request merged**